dev_guide/builds/build_inputs.adoc

```diff
@@ -196,6 +196,9 @@ specific commit, the system will fallback to a regular Git clone operation and
 checkout the commit, because using the `--depth=1` option only works with named
 branch refs.
 
+If the `ref` field denotes a pull request, the system will utilize a Git fetch operation
```

Comment:
certainly I can do utilize -> use 

but a few lines up "a regular Git clone operation" is used.  I was trying to be consistent.

How do you want to reconcile that @ahardin-rh ? thanks